src/routes/productRouter.js

```diff
@@ -2,7 +2,9 @@ const express = require("express");
 const router = express.Router();
 
 const productController = require("../controllers/productController");
+const { validateToken } = require("../utils/auth");
 
+router.get("/productId/:productId", productController.getProductDetail);
```

Comment:
```suggestion
router.get("/:productId", productController.getProductDetail);
```
엔드포인트 중간에 작성되는 부분은 없어도 될 것 같습니다